lib/helper/helper.go

```diff
@@ -84,10 +87,145 @@ func SaveSession(filename string, config structs.Configuration) error {
 
 // PrintSTAK prints out the short term access keys for AWS auth.
 func PrintSTAK(w io.Writer, stak kion.STAK, region string) error {
+	// handle windows vs linux for exports
+	var export string
+	if runtime.GOOS == "windows" {
+		export = "SET"
+	} else {
+		export = "export"
+	}
+
+	// conditionally print region
 	if region != "" {
 		fmt.Fprintf(w, "export AWS_REGION=%v\n", region)
 	}
-	fmt.Fprintf(w, "export AWS_ACCESS_KEY_ID=%v\nexport AWS_SECRET_ACCESS_KEY=%v\nexport AWS_SESSION_TOKEN=%v\n", stak.AccessKey, stak.SecretAccessKey, stak.SessionToken)
+
+	// print the stak
+	fmt.Fprintf(w, "%v AWS_ACCESS_KEY_ID=%v\nexport AWS_SECRET_ACCESS_KEY=%v\nexport AWS_SESSION_TOKEN=%v\n", export, stak.AccessKey, stak.SecretAccessKey, stak.SessionToken)
+
+	return nil
+}
+
+// SaveAWSCreds saves the short term access keys for AWS auth to the users AWS
+// configuration file.
+func SaveAWSCreds(stak kion.STAK, car kion.CAR) error {
```

Comment:
Suggested replacement of this long-term is to create a `credentials` process subsystem, generate config profiles for that and use the external process for credentials. Because of the volume needed, I think we need caching first. https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-sourcing-external.html